test/core/Container.js

```diff
@@ -352,6 +358,175 @@ describe('PIXI.Container', () =>
+        {
+            let container = new PIXI.Container();
+            let child = new PIXI.DisplayObject();
+
+            container.addChild(child);
+            container.destroy({ children: true });
+
+            expect(container.children.length).to.be.equals(0);
+            expect(container.transform).to.be.null;
+            expect(child.transform).to.be.null;
+
+            container = new PIXI.Container();
+            child = new PIXI.DisplayObject();
+
+            container.addChild(child);
+            container.destroy(true);
+
+            expect(container.children.length).to.be.equals(0);
+            expect(container.transform).to.be.null;
+            expect(child.transform).to.be.null;
+        });
+    });
+
+    describe('width', () =>
+    {
+        it('should reflect scale', () =>
+        {
+            const container = new PIXI.Container();
+            const graphics = new PIXI.Graphics();
+
+            graphics.drawRect(0, 0, 10, 10);
+            container.addChild(graphics);
+            container.scale.x = 2;
+
+            expect(container.width).to.be.equals(20);
+        });
+
+        it('should adjust scale', () =>
+        {
+            const container = new PIXI.Container();
+            const graphics = new PIXI.Graphics();
+
+            graphics.drawRect(0, 0, 10, 10);
+            container.addChild(graphics);
+
+            container.width = 20;
+
+            expect(container.width).to.be.equals(20);
+            expect(container.scale.x).to.be.equals(2);
+        });
+
+        it('should reset scale', () =>
+        {
+            const container = new PIXI.Container();
+
+            container.scale.x = 2;
+            container.width = 5;
+
+            expect(container.width).to.be.equals(0);
+            expect(container.scale.x).to.be.equals(1);
+        });
+    });
+
+    describe('height', () =>
+    {
+        it('should reflect scale', () =>
+        {
+            const container = new PIXI.Container();
+            const graphics = new PIXI.Graphics();
+
+            graphics.drawRect(0, 0, 10, 10);
+            container.addChild(graphics);
+            container.scale.y = 2;
+
+            expect(container.height).to.be.equals(20);
+        });
+
+        it('should adjust scale', () =>
+        {
+            const container = new PIXI.Container();
+            const graphics = new PIXI.Graphics();
+
+            graphics.drawRect(0, 0, 10, 10);
+            container.addChild(graphics);
+
+            container.height = 20;
+
+            expect(container.height).to.be.equals(20);
+            expect(container.scale.y).to.be.equals(2);
+        });
+
+        it('should reset scale', () =>
+        {
+            const container = new PIXI.Container();
+
+            container.scale.y = 2;
+            container.height = 5;
+
+            expect(container.height).to.be.equals(0);
+            expect(container.scale.y).to.be.equals(1);
```

Comment:
Yeah I dunno - I wrote the reset tests to cover [this functionality](https://github.com/pixijs/pixi.js/blob/185b8db01c268bb06161458a1c8b1ac32b5ab6d2/src/core/display/Container.js#L547-L550).

What should I do?